DCOS/dcos-testing.sh

```diff
@@ -312,6 +341,7 @@ test_windows_marathon_app() {
         "${APP_NAME}.marathon.mesos.thisdcos.directory"
         "${APP_NAME}.marathon.slave.mesos.thisdcos.directory"
     )
+    sleep 30
```

Comment:
Why this `sleep 30` ?

Reply:
this is a mistake that got pushed. I will remove it. It was there when i was having dns issues and i tried to see if it is not a timing issue. Will remove it ASAP